### DISCUSSION TITLE
python range vs enumerate

### DISCUSSION THREAD

**Discussion by Moreless91** [CloudBytes-Academy/CloudBytes.dev#110] [General]
https://cloudbytes.dev/snippets/python-for-loops-using-range-vs-enumerate

Your final examples are both range. One should be enumerate.

**Comment by rehanhaider** [comment]
Thanks, Apparently I copied pasted incorrect snippet. Will update.

**Comment by rehanhaider** [comment]
This has been updated now. Thanks for pointing out the error.
